bool go2takeoff

---

bool success
